public/docs/_examples/quickstart/ts/index.html

```diff
@@ -1,37 +1,27 @@
 <!DOCTYPE html>
-<!-- #docregion -->
 <html>
   <head>
-    <title>Angular QuickStart</title>
+    <title>Hello Angular</title>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" href="styles.css">
+    <style>
+      body {color:#369;font-family: Arial,Helvetica,sans-serif;}
+    </style>
 
-    <!-- 1. Load libraries -->
-    <!-- #docregion libraries -->
-    <!-- #docregion polyfills -->
-     <!-- Polyfill for older browsers -->
+    <!-- Polyfills for older browsers -->
     <script src="node_modules/core-js/client/shim.min.js"></script>
-    <!-- #enddocregion polyfills -->
 
     <script src="node_modules/zone.js/dist/zone.js"></script>
     <script src="node_modules/reflect-metadata/Reflect.js"></script>
     <script src="node_modules/systemjs/dist/system.src.js"></script>
-    <!-- #enddocregion libraries -->
-
-    <!-- 2. Configure SystemJS -->
-    <!-- #docregion systemjs -->
     <script src="systemjs.config.js"></script>
     <script>
       System.import('app').catch(function(err){ console.error(err); });
     </script>
-    <!-- #enddocregion systemjs -->
   </head>
 
-  <!-- 3. Display the application -->
-  <!-- #docregion my-app -->
   <body>
-    <my-app>Loading...</my-app>
+    <my-app><!-- content managed by Angular --></my-app>
```

Comment:
The **_content_** of the root component element is actually **completely ignored**, and so in that sense it would be more accurate to write "content is _not_ managed by Angular" :).  Maybe the intent was to write that the content **_generated_** by the Angular app gets inserted in the DOM below the root component element? So the remark could be changed to:
```
<my-app><!-- app generated content gets inserted here --></my-app>
```
Or
```
<my-app><!-- app generated content will appear here --></my-app>
```
If this is updated, then `index.1.html` will need to be updated (actually, it would be nice to be able to get rid of `index.1.html` altogether).